setup.py

```diff
@@ -31,7 +31,7 @@
     'six >= 1.10.0',
     'numpy >= 1.13.3',
     'decorator',
-    'cloudpickle == 1.3',  # TODO(b/155109696): Unpin cloudpickle version.
+    'cloudpickle >= 1.5.0',
```

Comment:
`from cloudpickle import CloudPickler` has always been the canonical way to import the `CloudPickler` class and should work with old the older versions. So it's possible to relax this to:

```suggestion
    'cloudpickle >= 1.3',
```